lua/wikis/commons/Condition.lua

```diff
@@ -76,28 +105,26 @@ local ConditionNode = Class.new(_ConditionNode,
 
 ---@return string
 function ConditionNode:toString()
+	if isBasicComparator(self.comparator) then
+		return String.interpolate(
+			'[[${name}${comparator}${value}]]',
+			{
+				name = self.name:toString(),
+				comparator = self.comparator,
+				value = self.value
+			}
+		)
+	end
 	return String.interpolate(
```

Comment:
Not sure I like this implementation, it's not very adaptable for potential future needs. What hjp suggests would be better